.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish Release
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  # unit tests
+
+  # package test
+  test:
+    runs-on: ${{ matrix.platform == 'linux' && 'ubuntu-latest' || matrix.platform == 'macos' && 'macos-13' || matrix.platform == 'windows' && 'windows-latest' }}
```

Comment:
do we need this? We only use `linux` platform. Could probably ax the whole `strategy` section

Reply:
ya, there's very few difference cross-platform in php and we're not in any of those niches.